src/overlords/mining/miner.ts

```diff
@@ -97,6 +110,52 @@ export class MiningOverlord extends Overlord {
 		}
 	}
 
+	checkIfDoubleMine() {
+		if (Game.rooms[this.pos.roomName]) {
+			// console.log('Double mining check');
+			this.source = _.first(this.pos.lookFor(LOOK_SOURCES));
```

Comment:
source is already computed in `populateStructures()`

Reply:
This needs an overhaul for performance. It works, but I turned it off because my suspicion is that the recalculation across all sources of double source every ~20 ticks isn't worth the tradeoff. Worth a glance later.

Reply:
This.source can be undefined so I'll leave an or as a safety